### PR TITLE
Add email parsing to Invite Colleagues

### PR DESCRIPTION
**Pull request by markwooff**
Addresses issue #1242

### CODE REVIEW

File: mod/invitefriends/scripts/multiple-emails.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+			checkDupEmail: true,
+			theme: "Bootstrap",
+			position: "top"
+		};
+		
+		// Merge send options with defaults
+		var settings = $.extend( {}, defaults, options );
+		
+		var deleteIconHTML = "";
+		if (settings.theme.toLowerCase() == "Bootstrap".toLowerCase())
```
Comment (nschonni) [inline mod/invitefriends/scripts/multiple-emails.js]: A switch/case might be better for this set of conditionals

File: mod/invitefriends/scripts/multiple-emails.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+				$(this).removeClass('multiple_emails-error');
+				var input_length = $(this).val().length;
+				
+				var keynum;
+				if(window.event){ // IE					
+					keynum = e.keyCode;
+				}
+				else if(e.which){ // Netscape/Firefox/Opera					
+					keynum = e.which;
+                }
```
Comment (nschonni) [inline mod/invitefriends/scripts/multiple-emails.js]: Spaces to tabs

File: mod/invitefriends/scripts/multiple-emails.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+				
+				//if(event.which == 8 && input_length == 0) { $list.find('li').last().remove(); } //Removes last item on backspace with no input
```
Comment (nschonni) [inline mod/invitefriends/scripts/multiple-emails.js]: Delete

**Comment by Steph4104** [comment]
remove_icon

**Comment by markwooff** [comment]
Ah that's the Font Awesome icon issue, on IE it will block font icons on GCcollab. GCconnex doesn't have this issue fortunately so they should appear when tested there.

**Pull request merged**